components/Article/TopImage/TopImage.js

```diff
@@ -0,0 +1,107 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import * as S from './TopImage.styled';
+
+const Container = ({children, featured, mb, value}) => {
+  if(featured)
```

Comment:
Aí força o trator né kkkkkk

Vamos colocar a abertura e fechamento desse if?

```suggestion
  if (featured) {
```

Reply:
Qual é o problema nessa simplificação? Pra mim tudo ok

Reply:
Para mim é invenção de moda 😂